lib/engine/config.ex

```diff
@@ -124,6 +124,10 @@ defmodule Engine.Config do
     updater = Application.get_env(:realtime_signs, :external_config_getter)
 
     case updater.get_active_headend_ip() do
+      {:ok, ""} ->
```

Comment:
The thought here is to add a new file to the `mbta-signs-dev` bucket but have the value for the headend IP as `""`

Reply:
Could we give the new environment its own separate bucket? That way we could keep everything isolated.

Reply:
I had planned to do it that way at first, but the infra team advised to reuse an existing bucket

Reply:
In that case, if we make the json value `null` instead of `""`, the existing code path should work without modifications.